BUILD.bazel

```diff
@@ -0,0 +1,407 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Description:
+#   Bazel build file for the ILM OpenEXR libraries.
+
+# Notes:
+# Copied from https://github.com/googlevr/seurat/blob/master/third_party/openexr.BUILD
+# Did some modifications to work with this OpenEXR version: https://github.com/Vertexwahn/openexr
+# (fork from master of OpenEXR around 28. September 2020)
+
+package(
+    default_visibility = ["//visibility:public"],
+    features = [
+        "-layering_check",
+        "-parse_headers",
+    ],
+)
+
+config_setting(
+    name = "linux_x86_64",
+    values = {"cpu": "k8"},
+    visibility = ["//visibility:public"],
+)
+
+config_setting(
+    name = "windows_x86_64",
+    values = {"cpu": "x64_windows"},
+    visibility = ["//visibility:public"],
+)
+
+DEFINES = [
+    "DISABLE_GOOGLE_GLOBAL_USING_DECLARATIONS",
+] + select({
+    ":linux_x86_64": [
+        "OS_LINUX=OS_LINUX",
+        "ARCH_K8",
+    ],
+    # The following match the build environment we've used to build Seurat
+    # libraries for Windows.
+    ":windows_x86_64": [
+        "NOGDI",  # Don't pollute with GDI macros in windows.h.
+        "NOMINMAX",  # Don't define min/max macros in windows.h.
+        "OS_WINDOWS=OS_WINDOWS",
+        "PRAGMA_SUPPORTED",
+        "WIN32",
+        "WINVER=0x0600",
+        "_CRT_SECURE_NO_DEPRECATE",
+        "_WIN32",
+        "_WIN32_WINNT=0x0600",
+        "_WINDOWS",
+
+        # Defaults for other headers (along with OS_WINDOWS).
+        "COMPILER_MSVC",
+
+        # Use math constants (M_PI, etc.) from the math library
+        "_USE_MATH_DEFINES",
+
+        # Allows 'Foo&&' (e.g., move constructors).
+        "COMPILER_HAS_RVALUEREF",
+
+        # Doublespeak for "don't bloat namespace with incompatible winsock
+        # versions that I didn't include".
+        # http://msdn.microsoft.com/en-us/library/windows/desktop/ms737629.aspx
+        "WIN32_LEAN_AND_MEAN=1",
+        "PLATFORM_WINDOWS",
+    ],
+})
+
+# Options for everything below
+all_options = [
+    "-D_THREAD_SAFE",
+] + select({
+    ":windows_x86_64": ["/EHsc"],
+    ":linux_x86_64": [
+        "-fexceptions",
+        "-fno-common",
+        "-Wno-unused-variable",
+        "-Wno-unused-result",
+        #"-Wno-string-conversion",
+        "-Wno-switch",
+        "-Wno-tautological-compare",
+        #"-Wno-deprecated-register",
+        # IlmImf/ImfHuf.cpp:111 wants to put a 21520 byte object on the stack.
+        "$(STACK_FRAME_UNLIMITED)",
```

Comment:
Is $(STACK_FRAME_UNLIMITED) a variable Bazel understands to expand? It is worth opening a second issue that ImfHuf.cpp creates a large object on the stack.